frontend/packages/policy-editor/src/utils/PolicyEditorUtils.test.ts

```diff
@@ -115,3 +121,128 @@ describe('PolicyEditorUtils', () => {
     });
   });
 });
+
+const mockActionId1: string = 'action1';
+const mockActionId2: string = 'action2';
+const mockActionId3: string = 'action3';
+
+const mockPolicyRulesActions: PolicyRule[] = [
+  {
+    ruleId: '1',
+    description: '',
+    subject: [],
+    actions: [mockActionId1, mockActionId2],
+    resources: [],
+  },
+  {
+    ruleId: '2',
+    description: '',
+    subject: [],
+    actions: [mockActionId3],
+    resources: [],
+  },
+];
+const mockPolicyActions: PolicyAction[] = [
+  { actionId: mockActionId1, actionTitle: 'Action 1', actionDescription: '' },
+  { actionId: mockActionId2, actionTitle: 'Action 2', actionDescription: '' },
+];
+
+const mockSubjectId1: string = 'subject1';
+const mockSubjectId2: string = 'subject2';
+const mockSubjectId3: string = 'subject3';
+const mockSubjectSourceRolecode: string = 'urn:altinn:rolecode';
+const mockSubjectSourceRolecodeWithoutUrn: string = 'altinn:rolecode';
+const mockSubjectSourceOrg: string = 'urn:altinn:org';
+
+const mockPolicySubject1: PolicySubject = {
+  subjectId: mockSubjectId1,
+  subjectTitle: 'Subject 1',
+  subjectSource: mockSubjectSourceRolecode,
+  subjectDescription: '',
+};
+const mockPolicySubject2: PolicySubject = {
+  subjectId: mockSubjectId2,
+  subjectTitle: 'Subject 2',
+  subjectSource: mockSubjectSourceOrg,
+  subjectDescription: '',
+};
+const mockPolicySubjects: PolicySubject[] = [mockPolicySubject1, mockPolicySubject2];
+
+const mockSubjectString1: string = `${mockSubjectSourceRolecode}:${mockSubjectId1}`;
+
+const mockPolicyRulesSubject: PolicyRule[] = [
+  {
+    ruleId: '1',
+    description: '',
+    subject: [mockSubjectString1, `${mockSubjectSourceOrg}:${mockSubjectId2}`],
+    actions: [],
+    resources: [],
+  },
+  {
+    ruleId: '2',
+    description: '',
+    subject: [`${mockSubjectSourceRolecode}:${mockSubjectId3}`],
+    actions: [],
+    resources: [],
+  },
+];
+describe('policyTabUtils', () => {
```

Comment:
I think this should be renamed as we are no longer in the ```policyTabUtils``` file. 
Also, we should have one outer describe for the file, and then a describe for each function being tested. 
So for this file, the outer describe should be ```describe('PolicyEditorUtils', () => {``` as it is on line 40, and then the describes for ```mergeActionsFromPolicyWithActionOptions``` and ```mergeSubjectsFromPolicyWithSubjectOptions``` should be inside it 😄